post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -1,51 +1,232 @@
+    2. Click **Roles** on the left.
+    3. In the main pane, click **Create role**.
+    4. Under **Select type of trusted entity**:
+        - Select the **AWS service** box.
+        - Click the  **Lambda** service from the list.
+        - Click **Next: Permissions**.
+    5. In the search box:
+        - Search for ` AWSLambdaBasicExecutionRole`.
+        - Select its check box.
+        - Search for `<YOUR_FSS_LAMBDA_POLICY>` where `<YOUR_FSS_LAMBDA_POLICY>` is the name of your custom File Storage Security policy. Example: `FSS_Lambda_Policy`.
+        - Select its check box in the list.
+        - You now have two policies selected.
+        - Click **Next: Tags**.
+        - (Optional) Enter tags.
+        - Click **Next: Review**.
+    6. On the **Review** page:
+        - In the **Role name** field, enter a name. Example: `FSS_Lambda_Role`.
+        - Make sure that two policies are listed.
+        - Click **Create role**.
+    </details>
+    
+   <details>
+   <summary>Using the AWS CLI</summary>
+
+    1. Enter the following AWS CLI command to create the role:
+
+        `aws iam create-role --role-name <YOUR_FSS_LAMBDA_ROLE>`
+
+        where `<YOUR_FSS_LAMBDA_ROLE>` is replaced with the name you want to give to the role. Example: `FSS_Lambda_Role`.
+    2. Attach the `AWSLambdaBasicExecutionRole` managed policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Role --policy-arn arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole`
+
+    3. Attach the custom policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Role --policy-arn <YOUR_FSS_LAMBDA_POLICY_ARN>`
+
+        where `<YOUR_FSS_POLICY_ARN>` is replaced with the File Storage Security custom policy's ARN that you noted earlier. Example: `arn:aws:iam::0123456789012:policy/FSS_Lambda_Policy`.
+    </details>
+
+## Deploy the Lambda
+
+<details>
+<summary>Using the AWS console</summary>
+
+1. **Create an empty function**
+    - Go to **Services > Lambda**.
+    - Click **Create function**.
+    - Select the **Author from scratch** box.
+    - In the **Function name** field, enter a name. Example: `FSS_Prom_Quar_Lambda`.
+    - From the **Runtime** drop-down list, select **Python 3.8**.
+    - Under **Permissions**, expand **Choose or create an execution role**.
+    - Select **Use an existing role**.
+    - In the drop-down list, select the execution role you created earlier. Example: `FSS_Lambda_Role`.
+    - Click **Create function** and leave the page open.
+2. **Add function code**
+    - Download the 'Promote or quarantine' [Handler.py file from GitHub](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-promote-or-quarantine/handler.py).
+    - On the AWS console page you left open, in the **Function code** section, remove the sample Lambda function code and paste the code from `handler.py`.
+    - Click **Save** at the top.
+</details>
```

Comment:
Here we miss a step.

After `Add function code` step, the user should add two environment variables, `PROMOTEBUCKET` and `QUARANTINEBUCKET`, on the AWS console page.
The values are the names of promote and quarantine buckets.

![image](https://user-images.githubusercontent.com/67313382/87617342-e06cfb00-c749-11ea-8e57-b6da893ca5d4.png)